src/message/messageSelectors.js

```diff
@@ -18,3 +18,6 @@ export const getAnchorForActiveNarrow = (narrow: Narrow) =>
     getMute,
     (messages, flags, subscriptions, mute) => findAnchor(messages, flags, subscriptions, mute),
   );
+
+export const getMessageById = (state: GlobalState, messageId: number): Message =>
```

Comment:
Not sure if we can call this a selector.
Also `state.messages[messageId]` seems simple and straightforward.
Why did you decide to extract it to a function?

Reply:
Because it looks/feels good to have similar selectors at same place. :)

Reply:
Can confirm, not a selector :)